bindings/vue/src/internal/optionsObjectHelper.js

```diff
@@ -58,7 +59,7 @@ const createComputedPropertiesFor = (targetClass) => {
       // register a computed property
       // which relay set/get operations to its corresponding property of DOM element
       computed[propertyName] = {
-        get() { return this.$el[propertyName]; },
+        get() { return this.$el[propertyName].__vue__ || this.$el[propertyName]; },
```

Comment:
`this.$el[propertyName].__vue__` throws an exception if `this.$el[propertyName]` is null. This should be fixed